droidlet/perception/robot/tests/utils.py

```diff
@@ -23,7 +24,8 @@ def init_physical_interfaces(self):
 
     def init_perception(self):
         self.perception_modules = {}
-        self.perception_modules["language_understanding"] = NSPQuerier(self.opts, self)
+        self.perception_modules["language_understanding"] = None
```

Comment:
was commenting this out intentional or just meant for local testing ?

Reply:
it was just for local testing, I guess I forgot to fix this one.   but!  I think this whole object is dead code, we are not ever using this FakeAgent